packages/client/src/components/MobileNavigation/MobileNavigation.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './MobileNavigation.css';
+
+export const MobileNavigation = (props) => {
+  const isOpenClass = props.open
+    ? 'mobile-navigation open'
+    : 'mobile-navigation';
+  return (
+    <nav className={isOpenClass}>
+      <Link to="/about-toolbox">about toolbox</Link>
+      <Link to="/contact-us">contact us</Link>
+    </nav>
+  );
```

Comment:
So where does the login/profile info go in mobile view?

Reply:
It stays in navigation as on pc view. This MobileNavigation component is needed for open/close menu in mobile view and has only these two links related to design.